src/styles/main.scss

```diff
@@ -1,3 +1,79 @@
+@import "./block/varribles";
+@import "./block/nav";
+@import "./block/header";
+@import "./block/cards";
+@import "./block/stars";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  font-family: "Roboto", sans-serif;
+}
+
+.catalog {
+  display: grid;
+
+  &__card {
+    transition: transform $transition-duration;
+    transform: scale(1);
+
+    &:hover {
+      transform: scale(1.2);
+    }
+  }
+}
+
+.catalog__content {
+  padding: 50px 0 72px;
+  display: grid;
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+}
+
+@media (min-width: 320px) {
+  .catalog__content {
+    grid-template-columns: repeat(1, 200px);
+    grid-template-rows: repeat(2, 408px);
+  }
+}
+
+@media (min-width: 488px) {
+  .catalog__content {
+    grid-template-columns: repeat(2, 200px);
+    grid-template-rows: repeat(2, 408px);
```

Comment:
Media queries can be nested inside a selector to avoid duplicating the class name several times